src/index.js

```diff
@@ -0,0 +1,68 @@
+for (let i = 30; i < 80; i += 5) {
+  document.body.onload = addElement(i);
+}
+
+/* a function to create the temps, 
+this is the only function syntax i could get to work?
+It didnt like when I tried an arrow function.
+Overall its a pretty clunky function. I'm still in the 
+"just trying to get it to work" phase with JS
+*/
+
+function addElement(i) {
+  let section = document.createElement('section');
+  section.className = 'container';
+
+  if (i < 45) section.style.color = 'rgb(103, 128, 147)';
+  else if (i >= 45 && i <= 60) section.style.color = 'rgb(81, 136, 98)';
+  else if (i >= 65) section.style.color = 'rgb(255, 236, 27)';
+
+  let div = document.createElement('div');
+  div.className = 'content';
+  let h3 = document.createElement('h3');
```

Comment:
In general we should use `const` for elements since we will not overwrite these, but modify their attributes